src/measure_theory/measure/vector_measure.lean

```diff
@@ -465,9 +475,36 @@ end measure
 
 namespace vector_measure
 
+open measure
+
 section
 
-variables [measurable_space β]
+include m
+
+/-- A vector measure over `ℝ≥0∞` is a measure. -/
+def ennreal_to_measure (v : vector_measure α ℝ≥0∞) : measure α :=
+of_measurable (λ s _, v s) v.empty (λ f hf₁ hf₂, v.of_disjoint_Union_nat hf₁ hf₂)
+
+lemma ennreal_to_measure_apply {v : vector_measure α ℝ≥0∞}
+  {s : set α} (hs : measurable_set s) : ennreal_to_measure v s = v s :=
+by rw [ennreal_to_measure, of_measurable_apply _ hs]
+
+/-- The equiv between `vector_measure α ℝ≥0∞` and `measure α` formed by
+`measure_theory.vector_measure.ennreal_to_measure` and
+`measure_theory.measure.to_ennreal_vector_measure`. -/
+@[simps] def equiv_measure : vector_measure α ℝ≥0∞ ≃ measure α :=
```

Comment:
in this one, I don't think it is a good idea to have `include m` instead of a typeclass assumption: there is no way for Lean to determine which `measurable_space` structure should be used.

Reply:
Is my understanding correct that `ennreal_to_measure` can have implicit `measurable_space` since it can infer if from the `vector_measure` argument?

Reply:
Yes, exactly!